errors/manifest.json

```
@@ -563,16 +563,34 @@
           "path": "/errors/sharp-version-avif.md"
         },
         {
-          "title": "script-in-document-page",
-          "path": "/errors/no-script-in-document-page.md"
+          "path": "/errors/no-script-in-document-page.md",
+          "redirect": {
+            "destination": "/errors/no-script-in-document.md"
```

```suggestion
            "destination": "/errors/no-script-in-document"
```
The redirect destination can omit the `.md` extension